index.html

```diff
@@ -44,7 +44,12 @@
     </div>
     <div class="site-wrapper">
       <section class="fifth-setion">
-        <div class="fifth-section-item">2</div>
+        <div class="fifth-section-item">
+          <h2 class="fifth-section-title">gdyugaldygulyfvayvf</h2>
+          <p class="fifth-section-text">
+            gsdyugfyulgfyusvfuyfve efugeufy fi g fu efuefgaelvfjs
+          </p>
```

Comment:
Not a must or even should. But, it's better to use [lorem ipsum generator](https://www.lipsum.com/) tools to generate dummy text. Please keep that in mind for the future.
[Lorem ipsum wikipedia](https://en.wikipedia.org/wiki/Lorem_ipsum)

Reply:
Ok.